README.md

```diff
@@ -30,14 +30,14 @@ Because everyone and every team has specific preferences or setups I recommend d
 
 
 #### Why would you use it?
-* It allows for a loosely coupled system. Loosely coupled systems have tons of benefits but that topic is out of the scope of this article, here are some benefits to give you an idea why you'll love this in your code.
-    * Allow simultaneous work between developers who are responsible for different components (such as UI layer and core logic). In most cases the model is defined at the start of the project, once the model is know you can define a contract between the view and controller. Now the developers know how to communication between the controller and view is defined and thus don't need to know how it is implemented.
+* It allows for a loosely coupled architecture. Loosely coupled systems have tons of benefits but that topic is out of the scope of this article. Here are 2 benefits (among many more) to give you an idea why you'll love this in your code.
```

Comment:
express, teveel dezelfde woorden is lastig, zeker moeilijkere, system was ook niet echt specifiek genoeg voor mijn taste. 1 maal architecture zorgt ervoor da ge dat daarna zonder probs kunt verdergebruiken